src/dev/precommit_hook/get_files_for_commit.js

```diff
@@ -44,7 +44,7 @@ import { File } from '../file';
 export async function getFilesForCommit() {
   const simpleGit = new SimpleGit(REPO_ROOT);
 
-  const output = await fcb((cb) => simpleGit.diff(['--name-status', '--cached'], cb));
+  const output = await promisify(simpleGit.diff)(['--name-status', '--cached']);
```

Comment:
According to the docs, we should be able to do this without `promisify`, like this:
```suggestion
  const output = await simpleGit.diff(['--name-status', '--cached']);
```